tensorflow/core/api_def/base_api/api_def_UnravelIndex.pbtxt

```diff
@@ -21,9 +21,9 @@ An 2-D (or 1-D if indices is 0-D) tensor where each row has the
 same shape as the indices array.
 END
   }
-  summary: "Converts a flat index or array of flat indices into a tuple of"
+  summary: "Converts a flat index or array of flat indices into a tuple of
```

Comment:
I think this has to be a single line. Maybe it can be shortened?

While we're here, adding an example would be fantastic.

Reply:
Sounds good, will try to build TF locally first to see whether a one-liner is a strong requirement. Example will follow.

Reply:
Good news: `pbtxt` _does_ support multi-line string (just testing compilation, this was also mentioned here: https://github.com/protocolbuffers/protobuf/issues/1297#issuecomment-389637051 )

Reply:
Right. And we have our own extension of pbtxt using bash like syntax (<<END). But the summary should in any case be a single line.

Reply:
Fair enough. The last commit contains also an example.